aspnet/fundamentals/localization.rst

```diff
@@ -203,7 +203,16 @@ If you only pass in one of the two (``culture`` or ``ui-culture``), the query st
 CookieRequestCultureProvider
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Production apps will often provide a mechanism to set the culture with the ``ASPNET_CULTURE`` cookie. The sample app provides code to set the cookie for French and English cultures.
+In old versions of ASP.NET a production apps will often provide a mechanism to set the culture with the ``ASPNET_CULTURE`` cookie.
```

Comment:
I think it's good idea to leave old name in the doc, because if I'm dev of old APS.NET then I will search something like "asp.net core ASPNET_CULTURE", and I have to find this docs where it's clear wrote that you have to use new cookie instead of old one.